### PR TITLE
asterisk debug: remove the old table and add text

### PR DESCRIPTION
**Pull request by pc-m**
the old table was not in markdown format and it became unwieldy with all the
columns

### CODE REVIEW

**Review by sduthil — APPROVED** [review]

**Comment by chartrandf** [comment]
I trhink the merge-it tag is missing? and  need a rebase

**Comment by pc-m** [comment]
> I trhink the merge-it tag is missing? and need a rebase

darn, you're right. I rebased and used `auto-merge`

File: content/uc-doc/contributors/debug_asterisk.md [hunk 1]
```diff
@@ -7,55 +7,92 @@ title: Debugging Asterisk
...
+3. Install dependencies:
+    apt-get update
+    apt-get install gdb libc6-dbg
+    apt-get install -t wazo-${WAZO_VERSION} asterisk-dbg xivo-libsccp-dbg
+
+4. Switch back to the production mirror:
+    xivo-dist phoenix
+
+
+### XiVO >= 15.01
```
Comment (fblackburn1) [inline content/uc-doc/contributors/debug_asterisk.md:80]:
```suggestion
### XiVO >= 16.01
```

**Comment by pc-m** [comment]
recheck

**Comment by pc-m** [comment]
recheck

**Comment by rgarrigue** [comment]
recheck

**Pull request merged**